### ISSUE TITLE
Compatible with screen reader

### ISSUE THREAD

**Issue by pillehu** [PHPFusion/PHPFusion#1070]
I would like to use a theme with screen reader, but it is not accessible for the blind users (w3c-wcag).
It would be very usefull if PHP-fusion could also follow this pattern and work like this.

**Comment by meangczac** [comment]
We will assign this as a milestone for future updates.
